src/adapter/src/catalog/storage.rs

```diff
@@ -58,6 +58,7 @@ pub use stash::{
 };
 
 const USER_VERSION: &str = "user_version";
+const DEPLOY_GENERATION: &str = "deploy_generation";
```

Comment:
It might be cleaner re-use the existing `const` instead of defining a new one.

Reply:
done